### PR TITLE
[CFM-891] Changes my group block titles

### PR DESCRIPTION
**Pull request by DavidHernandez**
Issue: https://issuetracker.amplexor.com/jira/browse/CFM-891

### CODE REVIEW

**Comment by DavidHernandez** [comment]
## Screenshots
### User has access to see the groups
groups-logged
### Groups with no access are gone. Desktop picture too.
groups-desktop
### Tablet
groups-tablet
### Mobile
groups-mobile

File: project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/my_groups/my-groups.tpl.php [hunk 1]
```diff
@@ -9,7 +9,7 @@
 <div class="row">
   <div class="col-md-12 right">
     <div class="my-groups panel-pane">
-      <h2 class="pane-title"><?php print t('My Groups') ?></h2>
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/my_groups/my-groups.tpl.php]: @DavidHernandez 
on your screenshots I can see "My groups" but here it seems like this is totally different.

Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/my_groups/my-groups.tpl.php]: ps, if you need to change it then re-roll first

**Comment by DavidHernandez** [comment]
Updated screenshots:

### Desktop
groups-desktop
### Tablet
groups-tablet
### Mobile
groups-mobile

**Pull request merged**